UCLALibrary/primo_ve
# SYS-1385: Restructure CSS

Implements [SYS-1385](https://uclalibrary.atlassian.net/browse/SYS-1385).

This PR restructures our CSS to work around the behavior of the Ex Libris development environment, which automatically concatenates all `*.css` files in a view's `css` directory into `css/custom1.css`.  We had previously broken that file up for more modular development, but weren't aware of this behavior.  There's no easy way to override that, but fortunately it doesn't apply to CSS files in subdirectories.

With this change, `css/custom1.css` is once again the only CSS file in `css`.  Our own custom files are now in `css/ucla/*.css`; the only content in `css/custom1.css` is a series of `import` statements which load our files.

The other code which was in `css/custom1.css` is now in `css/ucla/main.css`.

This view has been uploaded to https://search.library.ucla.edu/discovery/search?vid=01UCS_LAL:PRIMO_REDESIGN

To test locally:
* **BE SURE TO STASH/COPY/PRESERVE ANY LOCAL WORK IN PROGRESS**
* Pull and checkout this branch.
* Make sure `vars.env` points to `01UCS_LAL-PRIMO_REDESIGN`
* Startup local environment.
* Confirm that
  * All works as expected
  * Editing files in `css/ucla/` does not cause unwanted changes to `custom1.css`


[SYS-1385]: https://uclalibrary.atlassian.net/browse/SYS-1385?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
